src/metainfo.jl

```diff
@@ -5,15 +5,15 @@ type MetaVariable
     datatype::DataType
     dimensions::Array{Any}
     description::UTF8String
-    unit::UTF8String
+    unit::Any
 end
```

Comment:
Given that this only works with strings right now, maybe we can leave this as type string for now? We could make it an ``AbstractString``, or better even just leave it for now?